src/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -10,54 +10,105 @@ bridged into a constraint `F3`-in-`S3` (supported by the internal model) using b
 it will choose bridge 1 as it allows to bridge `F`-in-`S` using only one bridge instead of two if it uses bridge 2 and 3.
 """
 mutable struct LazyBridgeOptimizer{OT<:MOI.ModelLike} <: AbstractBridgeOptimizer
-    model::OT   # Internal model
-    con_to_name::Dict{CI, String}
+    # Internal model
+    model::OT
+    # Bridged variables
+    variable_map::Variable.Map
+    var_to_name::Dict{MOI.VariableIndex, String}
+    name_to_var::Union{Dict{String, MOI.VariableIndex}, Nothing}
+    # Bridged constraints
+    constraint_map::Constraint.Map
+    con_to_name::Dict{MOI.ConstraintIndex, String}
     name_to_con::Union{Dict{String, MOI.ConstraintIndex}, Nothing}
-    # Constraint Index of bridged constraint -> Bridge.
-    # It is set to `nothing` when the constraint is deleted.
-    bridges::Vector{Union{Nothing, Constraint.AbstractBridge}}
-    # Constraint Index of bridged constraint -> Constraint type.
-    constraint_types::Vector{Tuple{DataType, DataType}}
-    # For `SingleVariable` constraints: (variable, set type) -> bridge
-    single_variable_constraints::Dict{Tuple{Int64, DataType}, Constraint.AbstractBridge}
-    bridgetypes::Vector{Any} # List of types of available bridges
-    dist::Dict{Tuple{DataType, DataType}, Int}      # (F, S) -> Number of bridges that need to be used for an `F`-in-`S` constraint
-    best::Dict{Tuple{DataType, DataType}, DataType} # (F, S) -> Bridge to be used for an `F`-in-`S` constraint
+    # Bellman-Ford
+    variable_bridge_types::Vector{Any} # List of types of available bridges
+    variable_dist::Dict{Tuple{DataType}, Int}      # (S,) -> Number of bridges that need to be used for constrained variables in `S`
```

Comment:
Nit: These comments break the line limit. They could be moved above the field names instead.